builtin/mainmenu/tab_local.lua

```diff
@@ -22,6 +22,9 @@ if enable_gamebar then
 	function current_game()
 		local last_game_id = core.settings:get("menu_last_game")
 		local game = pkgmgr.find_by_gameid(last_game_id)
+		if game == nil then
+			mm_texture.reset()
+		end
```

Comment:
shouldn't this rather happen in the place `current_game()` is first called?

Reply:
I guess. I mean, this should cover even if someone deletes a game folder when the client is open, but how much likely is it to happen? Let me know

Reply:
Never mind, done